qase-pytest/src/qaseio/pytest/plugin.py

```diff
@@ -214,6 +214,16 @@ def _get_title(self, item):
 
         if not title:
             title = item.originalname
+        if title:
```

Comment:
It allows using pytest param in the formatted title - the same way as for allure title
```
qase.title("Some title with value: {value}")
```